docs/conf.py

```diff
@@ -50,7 +50,7 @@
 
 # General information about the project.
 project = u'OMERO.iviewer guide'
-copyright = u'2019, Open Microscopy Environment'
+copyright = u'2019-2020, Open Microscopy Environment'
```

Comment:
```suggestion
copyright = u'2019-2021, Open Microscopy Environment'
```

Not sure, but just a suggestion, because it is a new year ?

Reply:
PR was opened in 2020 :-)